src/main/java/edu/iu/uits/lms/crosslist/service/CrosslistService.java

```diff
@@ -399,6 +399,9 @@ public FindParentResult processSisLookup(String sisSectionId) {
          return findParentResult;
       }
 
+      // this style of Canvas search is case-sensitive, so let's force the upper case that will be accurate for SIS
+      sisSectionId = sisSectionId.toUpperCase();
+
       Section section = sectionService.getSection(String.format("sis_section_id:%s", sisSectionId));
```

Comment:
Maybe just do the toUpperCase on the line it actually gets used instead of reassigning to the variable?